client/client_priv.h

```diff
@@ -93,6 +93,7 @@ enum options_client
   OPT_REPORT_PROGRESS,
   OPT_SKIP_ANNOTATE_ROWS_EVENTS,
   OPT_SSL_CRL, OPT_SSL_CRLPATH,
+  OPT_QPS_TARGET,
```

Comment:
To keep in theme with the other SLAP arguments, we should name this OPT_SLAP_QPS_TARGET